ads/google/a4a/google-data-reporter.js

```diff
@@ -139,10 +139,10 @@ function setupPageLoadMetricsReporter_(ampElement) {
       'requests': {
         'fvt': 'https://csi.gstatic.com/csi?s=a4a' +
             `&c=${correlator}&met.a4a=` +
-	    'makeBodyVisible.${makeBodyVisible}~' +
```

Comment:
Why is this whitespace weird? It should be +4, right?

Reply:
Yeah, weird. Fixed so that the linter doesn't complain.